application/src/test/java/org/thingsboard/server/transport/lwm2m/Lwm2mTestHelper.java

```diff
@@ -55,6 +53,7 @@
     public static final int resourceId_2 = 2;
     public static final int resourceId_3 = 3;
     public static final int resourceId_4 = 4;
+    public static final int resourceId_7 = 7;
```

Comment:
should look like RESOURCE_ID_7 or RESOURCE_7

Reply:
RESOURCE_ID_7